scanpy/preprocessing/_scrublet/__init__.py

```diff
@@ -563,27 +537,21 @@ def scrublet_simulate_doublets(
 
     See also
     --------
-    :func:`~scanpy.external.pp.scrublet`: Main way of running Scrublet, runs
+    :func:`~scanpy.pp.scrublet`: Main way of running Scrublet, runs
         preprocessing, doublet simulation (this function) and calling.
-    :func:`~scanpy.external.pl.scrublet_score_distribution`: Plot histogram of doublet
+    :func:`~scanpy.pl.scrublet_score_distribution`: Plot histogram of doublet
         scores for observed transcriptomes and simulated doublets.
     """
-    try:
-        import scrublet as sl
-    except ImportError:
-        raise ImportError(
-            "Please install scrublet: `pip install scrublet` or `conda install scrublet`."
-        )
 
     X = _get_obs_rep(adata, layer=layer)
-    scrub = sl.Scrublet(X, random_state=random_seed)
+    scrub = Scrublet(X, random_state=random_seed)
 
     scrub.simulate_doublets(
         sim_doublet_ratio=sim_doublet_ratio,
         synthetic_doublet_umi_subsampling=synthetic_doublet_umi_subsampling,
     )
 
-    adata_sim = AnnData(scrub._E_sim)
+    adata_sim = AnnData(scrub._counts_sim)
```

Comment:
I am a bit of a sucker for `_xxx` as semantically indicating that they are private, but that is personal preference and I don't know what scanpy does in general off hand.  Again, the desire to make this PR small and minimal probably overrides this desire as there are other places where variable names did not change.  I do think then a refactor follow up would be good.

Reply:
I won't comment on every instance of these sorts of little variable things because it's out of scope.  But there are also these trailing underscores which I don't get.  Again, refactor is in order

Reply:
Yeah, the parts I changed were mostly to help myself understand. This example is unifying the naming scheme for some related class field names.

Reply:
> But there are also these trailing underscores which I don't get. 

https://scikit-learn.org/dev/developers/develop.html#estimated-attributes